tests/filesystem.go

```diff
@@ -253,7 +251,10 @@ func setupDisk(ctx context.Context, disk *types.Disk, diskIndex int, imageSize i
 			}
 		}()
 
-		partition.Device = fmt.Sprintf("%sp%d", disk.Device, partition.Number)
+		if partition.TypeCode == "blank" || partition.FilesystemType == "" || partition.FilesystemType == "swap" {
```

Comment:
What's the reason for moving this here, after the `Mkdir`?

Reply:
This might have caused the BB test failures.

Reply:
Was hoping this would just work…

The goal is to make sure that blank partitions end up with a Device and MountPath set, so that if it's later formatted, the tests know what to inspect.

I'll take a look at the failures.